tests/e2e/cht/pregnancy.spec.js

```diff
@@ -0,0 +1,136 @@
+    const activePregnancies = analyticsPo.targetById('active-pregnancies');
+    await helper.waitUntilReadyNative(activePregnancies);
+    const activeTitle = await analyticsPo.targetTitle(activePregnancies).getText();
+    const activeCount = await analyticsPo.targetNumber(activePregnancies).getText();
+    expect(activeTitle).toBe('Active pregnancies');
+    expect(activeCount).toBe('1');
+    await browser.get(utils.getBaseUrl() + 'contacts/' + pregnancyWoman._id);
+    await helper.clickElementNative(contactsPage.newActions);
+    await helper.clickElementNative(contactsPage.formById('pregnancy_home_visit'));
+    await helper.waitUntilReadyNative(genericFormPo.formTitle);
+    const pregnancyVisitReport = Factory.build('pregnancyVisit');
+    await formFiller(pregnancyVisitReport.fields, 'pregnancy_home_visit');
+    await helper.waitUntilReadyNative(activePregnancyCard);
+    [weeksPregnant, deliveryDate, ancVisit, lastVisit] = await contactsPage.cardChildrenValueArray(activePregnancyCard);
+    expect(weeksPregnant).toBe('34');
+    expect(deliveryDate).toBe(edd);
+    expect(ancVisit).toBe('1 of 8');
+    expect(lastVisit).toBe('today');
+    await helper.waitElementToDisappearNative(commonElements.snackBarContent);
+    await helper.clickElementNative(contactsPage.newActions);
+    await helper.clickElementNative(contactsPage.formById('delivery'));
+    await helper.waitUntilReadyNative(genericFormPo.formTitle);
+    const deliveryReport = Factory.build('delivery');
+    await formFiller(deliveryReport.fields, 'delivery');
+    const pastPregnancyCard = await contactsPage.cardElementByHeaderText('Past pregnancy');
+    await helper.waitUntilReadyNative(pastPregnancyCard);
+    const [dateDelivered,
+      placeOfDelivery,
+      babiesDelivered,
+      ancVisits] = await contactsPage.cardChildrenValueArray(pastPregnancyCard);
+    const expectDeliveryDate = moment().format(dateFormat);
+    expect(await dateDelivered).toBe(expectDeliveryDate);
+    expect(await placeOfDelivery).toBe('Health facility');
+    expect(await babiesDelivered).toBe('1');
+    expect(await ancVisits).toBe('1');
+    await commonElements.goToAnalytics();
+    const liveBirths = analyticsPo.targetById('births-this-month');
+    await helper.waitUntilReadyNative(liveBirths);
+    const liveBirthsCount = await analyticsPo.targetNumber(liveBirths).getText();
+    const liveBirthsTitle = await analyticsPo.targetTitle(liveBirths).getText();
+    expect(await liveBirthsCount).toBe('2');
+    expect(await liveBirthsTitle).toBe('Live births');
+    const inFacilityDeliveries = analyticsPo.targetById('facility-deliveries');
+    const inFacilityDeliveriesCount = await analyticsPo.targetNumber(inFacilityDeliveries).getText();
+    const inFacilityDeliveriesTitle = await analyticsPo.targetTitle(inFacilityDeliveries).getText();
+    expect(await inFacilityDeliveriesCount).toBe('100% (1 of 1)');
+    expect(await inFacilityDeliveriesTitle).toBe('In-facility deliveries');
```

Comment:
Why `await` each twice? I think this is equivalent and easier to read...

```suggestion
    const inFacilityDeliveriesCount = await analyticsPo.targetNumber(inFacilityDeliveries).getText();
    const inFacilityDeliveriesTitle = await analyticsPo.targetTitle(inFacilityDeliveries).getText();
    expect(inFacilityDeliveriesCount).toBe('100% (1 of 1)');
    expect(inFacilityDeliveriesTitle).toBe('In-facility deliveries');
```

This is done in a few places in this test, and I think we should be consistent about where we await so it's more obvious when one is missed.

Reply:
agreed, that's my bad. Have fixed in next commit.